test/e2e-tests/support/client_set.go

```diff
@@ -1,14 +1,49 @@
 package support
 
 import (
+	"github.com/tektoncd/pipeline/pkg/client/clientset/versioned"
+	v1 "github.com/tektoncd/pipeline/pkg/client/clientset/versioned/typed/pipeline/v1"
 	"k8s.io/client-go/kubernetes"
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/client-go/util/homedir"
+	knativetest "knative.dev/pkg/test"
 	"os"
 	"path/filepath"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+type Clients struct {
+	ControllerRuntime client.Client
```

Comment:
Yep I just left that in there by mistake was trying to use it but could't get the connection to the cluster to work. Looking at the `client.Create()` that is what I wanted instead of using each client.

Reply:
Removed.. for now atleast 👍

Reply:
Yeah, in the future we should either use the controller-runtime client or the dynamic client. It wasn't probably working for you because you didn't configure schemas for that connection? We can have a look at it in the future.